website/docs/reference/supported_commands.md

```diff
@@ -435,3 +435,48 @@ db.aggregate()
 |                   | `nameOnly`              | ❌     | [Issue](https://github.com/FerretDB/FerretDB/issues/301)  |
 |                   | `comment`               | ⚠️     | Ignored                                                   |
 |                   | `authorizedCollections` | ⚠️     | Ignored                                                   |
+
+## Diagnostic commands
+
+| Command           | Argument                | Status | Comments                                                  |
+|-------------------|-------------------------|--------|-----------------------------------------------------------|
+|`buildInfo`        |                            | ✅ | Basic command is fully supported                          |
+|                   | `version`                  | ✅ |                                                           |
+|                   | `gitVersion`               | ✅ |                                                           |
+|                   | `modules`                  | ✅ |                                                           |
+|                   | `sysInfo`                  |   | Deprecated                                                |
+|                   | `loaderFlags`              | ⚠️ | Ignored                                                   |
+|                   | `compilerFlags`            | ⚠️ | Ignored                                                   |
+|                   | `allocator`                | ⚠️ | Ignored                                                   |
+|                   | `versionArray`             | ✅ |                                                           |
+|                   | `openssl`                  | ⚠️ | Ignored                                                   |
+|                   | `javascriptEngine`         | ⚠️ | Ignored                                                   |
+|                   | `bits`                     | ✅ |                                                           |
+|                   | `debug`                    | ✅ |                                                           |
+|                   | `maxBsonObjectSize`        | ✅ |                                                           |
+|                   | `buildEnvironment`         | ✅ |                                                           |
+|                   | `ferretdbVersion`          | ✅ |                                                           |
+|                   | `ok`                       | ✅ |                                                           |
```

Comment:
Do we need to list `storageEngines`? https://www.mongodb.com/docs/manual/reference/command/buildInfo/#mongodb-data-buildInfo.storageEngines